docs/workshops/gerador-de-memes/README.md

```diff
@@ -0,0 +1,484 @@
+    ctx.font = `${tamanhoDaFonte}px Impact`;
+    ctx.lineWidth = tamanhoDaFonte / 20;
+
+    // Escrever texto de baixo
+    ctx.textBaseline = 'bottom';
+    textoBaixo.split('\n').reverse().forEach((t, i) => { // .reverse() porque estaremos escrevendo o texto de baixo a começar de baixo, para cima
+      ctx.fillText(t, canvas.width / 2, canvas.height - i * tamanhoDaFonte, canvas.width);
+      ctx.strokeText(t, canvas.width / 2, canvas.height - i * tamanhoDaFonte, canvas.width);
+    });
+  }
+
+  window.addEventListener('DOMContentLoaded', () => {
+    const textoCimaInput = document.getElementById('texto-cima');
+    const textoBaixoInput = document.getElementById('texto-baixo');
+    const tamanhoTextoCimaInput = document.getElementById('tamanho-texto-cima');
+    const tamanhoTextoBaixoInput = document.getElementById('tamanho-texto-baixo');
+    const imagemInput = document.getElementById('imagem-input');
+    const botaoGerar = document.getElementById('botao-gerar');
+
+    textoCimaInput.value = 'Valor de\nCima';
+    textoBaixoInput.value = 'Valor de\nBaixo';
+
+    botaoGerar.addEventListener('click', () => {
+      const leitor = new FileReader();
+      leitor.readAsDataURL(imagemInput.files[0]);
+
+      leitor.onload = () => {
+        const img = new Image();
+        img.src = leitor.result;
+        img.onload = () => {
+          gerarMeme(img, textoCimaInput.value, textoBaixoInput.value, tamanhoTextoCimaInput.value, tamanhoTextoBaixoInput.value);
+        };
+      };
+    });
+  });
+  ```
+  
+  </div>
+
+</details>
+
+**_E com isso, terminamos de programar nosso Gênio Meme! Confira o que você construiu clicando no botão "Run" no topo da página._**
+
+![Resultado final](img/resultado_final.gif)
+
+>Nota: Para salvar o meme gerado, você pode clicar com o botão direito no meme e selecionar "salvar como..."
+
+## Parte 4: Hackeando
+
+Agora que terminamos de construir um site bem básico, podemos adicionar um monte de coisas legais a ele!
+
+Aqui vão algumas sugestões do que você pode adicionar:
+
+1. Tente adicionar um [seletor de cores](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/input/color), para que o usuário possa mudar a cor do textoCima/textoBaixo.
+2. Adicione o recurso arrastar e soltar, utilizando a [API de Drag and Drop do HTML5](https://web.dev/drag-and-drop/).
+3. Utilize [Flexbox](https://developer.mozilla.org/pt-BR/docs/Learn/CSS/CSS_layout/Flexbox) criar um layout bacana.
+4. Utilize uma API a fim obter modelos para memes, como o disponibilizado pela [ImgFlip](https://imgflip.com/api).
+5. Adicione botões de baixar e compartilhar para mostrar a todos seu meme incrível!
+
+## Parte 5: O Fim
+
+Se você não criou uma conta no [repl.it](https://repl.it), o faça para que você consiga salvar essa sua maravilhosa criação!
+
+Se você está tendo problemas para se cadastrar, fale com um líder de seu clube ou com alguém no [Slack do Hack Club](https://hackclub.com/slack)!
```

Comment:
Colocar este texto na Parte 2 do documento, onde será pedido ao usuário criar uma conta no repl.it